src/assisted_test_infra/test_infra/controllers/node_controllers/oci_api_controller.py

```diff
@@ -348,9 +356,14 @@ def _apply_job_from_stack(
         # on success, we return the jobs output - list
         items = self._resource_manager_client.list_job_outputs(job.data.id).data.items
         for item in items:
-            if item.output_name == "oci_ccm_config":
+            if item.output_name == "dynamic_custom_manifest":
+                file_path = (
+                    f"/tmp/oci-{self._entity_config.cluster_id}/terraform-output-{self._entity_config.cluster_id}.yml"
```

Comment:
will it be ok it I upload the manifest directly to the AI in a next change: https://github.com/openshift/assisted-test-infra/pull/2573/files#diff-901cf347567b426cc7b9d878e54d1f44424c1b84cd94c5c444adb51fa6d0eebcR359 ?